src/python/pants/backend/codegen/protobuf/subsystem/protoc.py

```diff
@@ -0,0 +1,17 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.binaries.binary_tool import NativeTool
+
+
+class Protoc(NativeTool):
+  options_scope = 'protoc'
+  support_dir = 'bin/protobuf'
+  default_version = '2.4.1'
+
+  replaces_scope='gen.protoc'
```

Comment:
Can this be a literal `deprecated_options_scope`?

Also, the whitespace around `=` here is inconsistent.

Reply:
`deprecated_options_scope` means that all options can be read from the other scope. In the specific case of protoc that would work, because the old option was also named `--version`, but that need not be the case in general.  Once I've migrated all our uses, if it turns out we can get away with `deprecated_options_scope` then I'll remove this machinery.  Note that even if not, this is a just a temporary aide to migrating to this new scheme. We can delete it once that's achieved.